compiler/damlc/tests/daml-test-files/ActionFail.daml

```diff
@@ -1,7 +1,7 @@
 -- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @INFO range=9:21-9:38; Use catOptionals
+-- @IGNORE range=7:21-7:38; Use catOptionals
```

Comment:
You probably need to go back to `9` here.

Reply:
yes, and ignore should be info.